pkg/vere/io/http.c

```diff
@@ -613,6 +620,70 @@ _http_req_dispatch(u3_hreq* req_u, u3_noun req)
   }
 }
 
+
+
+/* _http_cache_respond(): respond with a simple-payload:http
+*/
+static void
+_http_cache_respond(u3_hreq* req_u, u3_noun nun) {
+  // XX check auth
+  u3_noun auth, response_header, data;
+  u3x_qual(u3t(u3t(nun)), &auth, 0, &response_header, &data);
```

Comment:
I think this function leaks `u3h(nun)` and `u3h(u3t(nun))` -- they're no longer referred to, but the refcount is still nonzero.  The easy way to fix this is to `u3k(u3t(u3t(nun)))` here when destructuring it, then at the end of the function, `u3z(nun)`.